code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -18,7 +18,7 @@
 	icon_state = "ai"
 	move_resist = MOVE_FORCE_OVERPOWERING
 	density = TRUE
-	canmove = FALSE
+	mobility_flags = ALL
```

Comment:
ALL and not NONE?

Reply:
AI stuff checks for mobility, AIs can't move even with MOBILITY_MOVE.
So many things in code just check for "can the user move" that it wasn't worth it at the time (and honestly not even now) to make a system to distinguish tbh (well this system kinda does but AIs still get MOBILITY_MOVE)